app/models.py

```diff
@@ -888,14 +888,14 @@ def apply(self, domain, post_records):
 
         # Adjustment to add multiple records which described in https://github.com/ngoduykhanh/PowerDNS-Admin/issues/5#issuecomment-181637576
         final_records = []
-        if NEW_SCHEMA:
-            records = sorted(records, key = lambda item: (item["name"], item["type"]))
-            for key, group in itertools.groupby(records, lambda item: (item["name"], item["type"])):
+        records = sorted(records, key = lambda item: (item["name"], item["type"]))
+        for key, group in itertools.groupby(records, lambda item: (item["name"], item["type"], item["changetype"], item["ttl"])):
```

Comment:
`item["ttl"]` shouldn't be in the `groupby`, otherwise this will split into 2 array (in case 2 records have same Name-Type but difference TTL) then one of them will be deleted. 

Tested on pdns 4.0.0~alpha2-3build1